Description: Drop valuable reference until rust-valuable-serde makes it through NEW
Author: John Goerzen <jgoerzen@complete.org>
Author: Peter Michael Green <plugwash@debian.org>
Last-Update: 2022-12-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: tracing-subscriber/Cargo.toml
===================================================================
--- tracing-subscriber.orig/Cargo.toml
+++ tracing-subscriber/Cargo.toml
@@ -210,23 +210,6 @@ std = [
     "alloc",
     "tracing-core/std",
 ]
-valuable = [
-    "tracing-core/valuable",
-    "valuable_crate",
-    "valuable-serde",
-    "tracing-serde/valuable",
-]
-
-[target."cfg(tracing_unstable)".dependencies.valuable-serde]
-version = "0.1.0"
-optional = true
-default-features = false
-
-[target."cfg(tracing_unstable)".dependencies.valuable_crate]
-version = "0.1.0"
-optional = true
-default-features = false
-package = "valuable"
 
 [badges.maintenance]
 status = "experimental"
